docs/extra/workers.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: Workers
+parent: Extra
+nav_order: 1
+---
+
+# Workers
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+  {:toc}
+
+Orka is exposing a useful factory to create workers. This is a convenient boilerplate code to execute long running tasks keeping some state in a mongodb collection.
+Sometimes you need to execute long running tasks that need to continue from where it had left off when being restarted. E.g. Processing some million of rows in a table and keeping the last processed row.
+Orka's worker will automatically create and use a mongo collection called workerjob for this.
```

Comment:
it will be useful to add an example of how this integrates with a cloud environmnet. ie, how we used it in k8s or how one can use let;s say using another cloud provider.

Reply:
Is our solution with production-env-configs something that makes sense outside workable? 
Also we currently don't have any cloud environment documentation for starting an orka enabled app.
Maybe we should discuss with @innnkkki if they are interested in contributing to the docs.